### ISSUE TITLE
New idea for /is settings

### ISSUE THREAD

**Issue by chudnoffsky** [tastybento/askyblockIssues#934]
Hello, dear **tastybento**!
I would like to offer you **an idea for your askyblock plugin on /is settings.** Now I use the latest version of the plugin - **ASkyBlock V3.0.9.4**                                                                                                          

The idea is for you to add more settings to /is settings. For example, disabling mobs, allowing you to change on the island of your time and weather, so that you have changed the time and weather, and others do not, the regulation of the spread of fire on the island, picking up things on the island, to regulate pvp on the island and explosions on the island. 
These settings are in your plugin is not enough, they will be very useful.
After, I had another idea, to make a special law where it would be possible to keep player items upon death, so you have created permission to do this, it can also be adjusted to include the preservation of the inventory or to turn it off! 

This is where my ideas run out, I hope you will realize soon. Waiting for your feedback, good luck!

**Comment by Yoolabc** [comment]
This idea normal. but me enough of these functions

**Comment by Poslovitch** [comment]
First things first : https://github.com/tastybento/ASkyBlock-Bugs-N-Features/blob/master/README.md

- disabling mobs: already exists;
- allowing you to change on the island of your time and weather: interesting, though I don't see the interest;
- spread of fire on the island: already exists;
- picking up things on the island: already exists;
- regulate pvp: already exists;
- explosions on the island: already exists;
- keep player items upon death: already exists;

So, TL;DR:
```
ASkyBlock & AcidIsland Issues

ASkyBlock and AcidIsland support has now ended! Issues will be looked at only on a best-effort basis or maybe not at all. Thank you for your support over the years, but we're now moving onto other projects.

The development team are now 100% working on BentoBox (BSkyBlock, AcidIsland)

ASkyBlock is NO LONGER ACCEPTING FEATURE REQUESTS
```

Aaaaaand what you suggested almost entirely already exists;
